sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Azure.AI.Language.QuestionAnswering.csproj

```diff
@@ -33,5 +33,8 @@
     <Compile Include="$(AzureCoreSharedSources)OperationInternalOfT.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)TaskExtensions.cs" LinkBase="Shared" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Azure.Core should almost always be a `PackageReference`.

Reply:
When regenerating the client using the updated Autorest.Csharp package, the client references an overload of the methods 'CreateMessage' and 'WaitForCompletion' Async that are not available in the Core package. There doesn't seem to be any updates to the Core package.

As such, the project does not build without this project reference. I believe the alternative would be to wait for a new Core package (I'm not sure how frequently it's released). 

By the way, I have included you in an email thread about a bug that prompted me to update Autorest.Csharp package in the first place.

Reply:
@pakrym what should the process be in this case? We've had a few over the past couple years so it's a bit of a jumble what's now correct.

Reply:
cc @annelo-msft

Reply:
@heaths, when we make changes to Azure.Core that are required by the client libraries, we move the libraries over to a project reference so they'll continue to build until the new Azure.Core package is released.  Once Azure.Core is released (usually a few days before the client library package releases), we change those back to package references.

Reply:
Didn't we have some properties that some Azure.Core.props file used to do that, or something like that? I don't see the file now, though. If this indeed is the consistent way I can sign off, but I know we had a different way of doing this once upon a time.

At the very least, @AhmedLeithy you should open a bug and add a TODO or (old school) BUGBUG comment referencing it to make sure it gets cleaned up. Accidentally shipping this SDK against a ProjectReference of Azure.Core can and has (for another SDK) caused issues.

Reply:
I've added a TODO comment and created [this Bug/Issue](https://github.com/Azure/azure-sdk-for-net/issues/26171) to remind us to change it before release: